Assets/Scripts/KeyBindings/KeyBindings.cs

```diff
@@ -43,4 +46,13 @@ public static bool SetKeyBinding(string key, KeyCode binding)
             return false;
```

Comment:
What is this?

Reply:
this is returning false in case the keyBinding could not be set